compiler/plugins/target/AMD-AIE/aie/AMDAIEDmaToNpu.cpp

```diff
@@ -376,6 +376,13 @@ struct DmaToNpuPattern : OpConversionPattern<NpuDmaMemcpyNdOp> {
     // repeat_count
     repeat_count = IntegerAttr::get(i32ty, sizes[3] - 1);
 
+    // enable_packet
+    if (auto packetInfo = op.getPacket()) {
```

Comment:
Where is the first place that packet control is turned on?

Reply:
E2E you mean? I have one hacked example working locally that enables it for L3 -> L2 data movement, but I am working on generalizing it.